src/Diagnostics.Logger/ETWProvider/DiagnosticsETWProvider.cs

```diff
@@ -451,6 +451,24 @@ public void LogKustoQueryInformation(string Source, string RequestId, string Mes
                 ExceptionDetails);
         }
 
+        /// <summary>
+        /// Log kusto query information.
+        /// </summary>
+        /// <param name="RequestId">Request id.</param>
+        /// <param name="Message">The message.</param>
+        /// <param name="ExceptionType">Exception type.</param>
+        /// <param name="ExceptionDetails">Exception details.</param>
+        [Event(3004, Level = EventLevel.Informational, Channel = EventChannel.Admin, Message = ETWMessageTemplates.LogKustoQueryInformation)]
+        public void LogKustoHeartbeatInformation(string RequestId, string Message, string ExceptionType, string ExceptionDetails)
+        {
+            WriteDiagnosticsEvent(
+                3005,
+                RequestId,
```

Comment:
Remove requestid since there is no request coming in from client. This will make it confusing for our other requestid